### PR TITLE
Added JQuery to the Linter File

### PR DESCRIPTION
**Pull request by sarj21**
This is necessary to keep editors happy

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -24,5 +24,11 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
     'import/no-unresolved': 0,
+    // Were developing on windows so this is necessary
+    'linebreak-style': ['error', 'windows'],
+  },
+  globals: {
+    "$": true,
+    "jQuery": true
```
Comment (mecolmg) [inline .eslintrc.js]: Add trailing comma

**Comment by sarj21** [comment]
Not sure if this will cause Mac issues. I can switch atom to unix line endings to solve the problem as well. Try adding that line to your editor and seeing if you get a bunch of errors. It tells me I have an error on every file with invalid line endings because the Airbnb file specifies Unix

**Comment by mecolmg** [comment]
It does cause errors on mine, but there's two options we can use to get past this quick:
- We could add `fix: true,` to the options for the eslinter in build/webpack.base.conf.js which will try to autofix linter errors, including this line-endings issue (https://github.com/MoOx/eslint-loader#fix-default-false)
or
- We could add `"fix": "eslint --ext .js,.vue --fix src"`, to the package.json file which would allow us to run `npm run fix` to fix errors instead on demand. 

I'm kind of leaning to the  autofix right now, as it'll ensure that the code is consistent across the entire repo, but up to debate.

**Comment by mecolmg** [comment]
I added both in this commit, we can leave both if you're fine with autofix, or I can amend and remove the autofix if you don't want it.

**Comment by sarj21** [comment]
This also should've been 2 separate requests. We will go ahead with autofix and I'll submit a new request for the jquery linting

**Pull request closed without merging**